websites/T/Tinkercad/metadata.json

```
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Tinkercad",
+	"description": {
+		"en": "Tinkercad is a free, easy-to-use app for 3D design, electronics, and coding"
+	},
+	"url": "www.tinkercad.com",
+	"matches": [
+		"*://www.tinkercad.com/*"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/XXez3yK.png",
```

Image appears to be removed, or a temporary issue by Imgur.
```suggestion
	"logo": "https://i.imgur.com/2Nyp2DT.png",
```